lib/node_modules/@stdlib/ndarray/base/mskfilter/docs/repl.txt

```diff
@@ -0,0 +1,44 @@
+
+{{alias}}( arrays )
+    Applies a mask to a provided input ndarray and assigns unmasked values to
+    elements in a provided one-dimensional output ndarray.
+
+    A provided "ndarray" should be an `object` with the following properties:
+
+    - dtype: data type.
+    - data: data buffer.
+    - shape: dimensions.
+    - strides: stride lengths.
+    - offset: index offset.
+    - order: specifies whether an ndarray is row-major (C-style) or column-major
+    (Fortran-style).
+
+    Parameters
+    ----------
+    arrays: ArrayLikeObject<ndarray>
+        Array-like object containing an input, mask and output ndarray.
+
+    Examples
+    --------
+    // Define ndarray data and meta data...
+    > var xbuf = new {{alias:@stdlib/array/float64}}( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
```

Comment:
I am not sure How can we make it to less than 80 characters.

Reply:
I have done the required changes and make the xbuf of length 7. But still it's exceed 80 characters limit. 
Should we update this examples of `2d ndarray` ?